challenge_opener/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="toolbar_challenge_opener_elevation">4dp</dimen>
+    <dimen name="scrollview_challenge_activity_opener_margin">56dp</dimen>
+    <dimen name="icon_help_size">32dp</dimen>
+    <dimen name="txtview_challenges_detail_description_txtsize">18sp</dimen>
+    <dimen name="txtview_challenges_detail_description_margin">8dp</dimen>
+    <dimen name="txtview_toolbar_title_textsize">20sp</dimen>
+    <dimen name="btn_challenges_opener_margin">8dp</dimen>
+    <dimen name="btn_recyclerview_challenge_cornersize">16dp</dimen>
+    <dimen name="btn_challenges_opener_txtsize">24sp</dimen>
+    <dimen name="togglebtn_challenges_detailopener_margin">8dp</dimen>
+    <dimen name="txtview_expandableview_activitydescription_margin">8dp</dimen>
```

Comment:
Yeah, however I think it would be best if that info were placed on the project's wiki, since that info is more specific than the general descriptions often found in README. @AlejoGonzalezU @mauriciohernandez-globant do you agree?

Reply:
@js-gonzalez @sonia-rodriguez-g Talking about resources, IMHO, I prefer snake cases and avoid camel cases. E.g.

```
<what>_<where>_<size(optional)>
radius_card_view_8_dp
```
The nomenclature is an agreement, so it's fine keeping this style if you feel comfortable with that.

Nevertheless, I totally agree that, as a good practice, you define a README file with information about your code.